cnf-tests/testsuites/e2esuite/fec/fec.go

```diff
@@ -0,0 +1,169 @@
+package fec
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	runtimeClient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	sriovfecv1 "github.com/open-ness/openshift-operator/sriov-fec/api/v1"
+
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/client"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/discovery"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/nodes"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/utils"
+)
+
+var (
+	nodeName string
+	numVfs   = 2
+)
+
+var _ = Describe("fec", func() {
+	BeforeEach(func() {
+		if discovery.Enabled() {
+			// TODO: change this to read the sriovFecClusterConfig and validate all the nodes
+			Skip("sriov-fec test doesn't support discovery mode")
+		}
+
+		err := cleanSriovFecClusterObject()
+		Expect(err).ToNot(HaveOccurred())
+
+		err = createSriovFecClusterObject()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	AfterEach(func() {
+		err := cleanSriovFecClusterObject()
```

Comment:
Do we need to clean before and after?

Reply:
Also, would it make sense to add these new resources/namespaces to the reporter?

Reply:
Also also, would it make sense to export a Clean function to be called when the suite shuts down?